packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/dataset/Dataset.java

```diff
@@ -32,6 +32,12 @@ public class Dataset extends TerariumAsset {
 	@TSOptional
 	private String userId;
 
+	/**
+	 * ESGF id of the dataset. This will be null for datasets that are not from ESGF
+	 **/
+	@TSOptional
+	private String esgfId;
```

Comment:
does this need to be "esgfId" or can it be more generic, like "rawId" or "originalId" or something like that?

Reply:
It could be anything, but, right now its specific only to the ESGF service. Worth changing?